plugins/org.jkiss.dbeaver.data.transfer/plugin.xml

```diff
@@ -154,6 +154,7 @@
                     <property id="quoteNever" label="%dataTransfer.processor.csv.property.quoteNever.name" type="boolean" description="%dataTransfer.processor.csv.property.quoteNever.description" defaultValue="false" required="false"/>
                     <property id="nullString" label="%dataTransfer.processor.csv.property.nullString.name" type="string" description="%dataTransfer.processor.csv.property.nullString.description" defaultValue="" required="false"/>
                     <property id="formatNumbers" label="%dataTransfer.processor.csv.property.formatNumbers.name" type="boolean" description="%dataTransfer.processor.csv.property.formatNumbers.description" defaultValue="false" required="false"/>
+                    <property id="formatArray" label="%dataTransfer.processor.csv.property.formatArray.name" type="string" description="%dataTransfer.processor.csv.property.formatArray.description" defaultValue="{}" required="true" validValues="{ },[ ],( )" allowCustomValues="false"/>
```

Comment:
Why is the default value different from valid values? Please make values consistent

Reply:
fixed!